pages/Documents/Reporting/OutboundConfigurationAPI/overview.md

```diff
@@ -0,0 +1,400 @@
+---
+pagename: Overview
+sitesection: Documents
+categoryname: Reporting
+documentname: Outbound Configuration API
+permalink: outbound-configuration-api-overview.html
+root-link: true
+indicator: Proactive Messaging
+redirect_from:
+ - proactive-messaging-reporting-api.html
+---
+
+### Introduction
+The Outbound configuration API provides a complete list of API handoffs created in the Proactive Web Tool. This API allows 1) Fetching all handoffs configurations 2) Fetching all handoff configurations filtered by a specific messaging channel (SMS, WhatsApp, Apple Business Chat, In-app, Google Rcs, etc.)
+
+API provides outbound configuration for below services
+* Proactive Messaging 2.0
+
+
+### Getting Started
```

Comment:
Please use sentence case for all headings (as per [style guide](https://github.com/LivePersonInc/developer-center#notes-on-content-and-code)).